# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp
# Commit: 302c30b598b0
# Full Hash: 302c30b598b0ed12b212c19f86256723dd362835
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-07 04:06:39
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r 6e2d679c7168 -r 302c30b598b0 gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp	Wed May 06 21:21:46 2020 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/renderer11_utils.cpp	Wed May 06 22:04:28 2020 +0000
@@ -2435,7 +2435,9 @@
             // Haswell drivers occasionally corrupt (small?) (vertex?) texture data uploads for
             // 128bit formats.
             features->setDataFasterThanImageUploadOn128bitFormats.enabled = false;
-        }
+        } else if (IsSandyBridge(adapterDesc.DeviceId)) {
+	    features->emulateClearViewAfterDualSourceBlending.enabled = true;
+	}
     }
 
     if (IsAMD(adapterDesc.VendorId))